validators/cmd/sigs/sigs-validator_test.go

```diff
@@ -0,0 +1,462 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package main
```

Comment:
It is highly recommended to place test files in their own (test) package in order to avoid testing internal implementations.

Reply:
These tests explicitly test an internal implementation, as it's essential. I believe this to be a valuable tradeoff.

WDYT?